node/node.go

```diff
@@ -107,16 +118,15 @@ type AlgorandFullNode struct {
 	lastRoundTimestamp    time.Time
 	hasSyncedSinceStartup bool
 
-	txPoolSyncer *rpcs.TxSyncer
+	txPoolSyncerService *rpcs.TxSyncer
```

Comment:
If you touch it... please move above to other services?

Reply:
sure.